Package.swift

```diff
@@ -33,6 +33,10 @@ let swiftPMDataModelProduct = (
     ]
 )
 
+#if compiler(>=5.5)
```

Comment:
I think some of the toolchain CI is still using an older Swift version, so that might be problematic.  But it would be great to start being able to rely on synthesized Codable of enums in particular, and that requires 5.5.